src/main/scala/ai/privado/languageEngine/python/processor/PythonProcessor.scala

```diff
@@ -52,38 +65,35 @@ object PythonProcessor {
         try {
           logger.info("Applying default overlays")
           logger.info("=====================")
-          println(
-            s"${TimeMetric.getNewTime()} - Run oss data flow is done in \t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
-          )
-
+          statsRecorder.initiateNewStage("Default overlays")
           // Apply default overlays
           X2Cpg.applyDefaultOverlays(cpg)
+          statsRecorder.endLastStage()
+          statsRecorder.initiateNewStage("Overriden passes")
           new ImportsPass(cpg).createAndApply()
           new PythonImportResolverPass(cpg).createAndApply()
           new PythonInheritanceNamePass(cpg).createAndApply()
-          println(
-            s"${TimeMetric.getNewTime()} - Run InheritanceFullNamePass done in \t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
-          )
-
-          new HTMLParserPass(cpg, sourceRepoLocation, ruleCache, privadoInputConfig = privadoInput)
```

Comment:
Looks like we removed this pass. We need to add it back. Looks like we made few changes for customer repo debugging, which we didn't revert back.